src/openforms/authentication/api/views.py

```diff
@@ -28,3 +30,20 @@ class PluginListView(ListMixin, APIView):
 
     def get_objects(self):
         return list(register)
+
+
+class AuthenticationLogoutView(GenericAPIView):
+    authentication_classes = ()
+    permission_classes = (permissions.AllowAny,)
+    serializer_class = (
+        serializers.Serializer
+    )  # just to shut up some warnings in drf-spectacular
+
+    @extend_schema(
+        summary=_("Delete session"),
+        description=_("Calling this endpoint will clear the current user session."),
+        responses={(204, "application/json"): ""},
+    )
+    def delete(self, request, *args, **kwargs):
+        request.session.flush()
+        return HttpResponse(status=status.HTTP_204_NO_CONTENT)
```

Comment:
```suggestion
    @extend_schema(
        summary=_("Delete session"),
        description=_("Calling this endpoint will clear the current user session and delete the session cookie."),
    )
    def delete(self, request, *args, **kwargs):
        request.session.flush()
        return HttpResponse(status=status.HTTP_204_NO_CONTENT)
```
For a `delete` call, the HTTP 204 is the default and doesn't need to be specified explicitly.